### DISCUSSION TITLE
Problema con SatHttpGetaway

### DISCUSSION THREAD

**Discussion by JuanJLM20** [phpcfdi/cfdi-sat-scraper#79] [Q&A]
Hola.. Me parece muy interesante este proyecto... Quise probarlo pero al ejecutar el archivo para probar, me salió el siguiente error: 
![image](https://github.com/phpcfdi/cfdi-sat-scraper/assets/146992708/f2bdebea-d977-4103-9b82-9f39ae689bb3)
Espero puedan ayudarme:)

**Comment by eclipxe13** [comment]
Este proyecto se debe poner como una dependencia en tu propio proyecto usando composer.
Sigue las instrucciones de instalación y uso.
